brave/src/main/java/brave/internal/baggage/BaggageFieldsFactory.java

```diff
@@ -51,7 +51,7 @@ public static BaggageFieldsHandler create(List<BaggageField> fields, boolean dyn
     this.initialArrayLength = initialState.length;
   }
 
-  @Override protected BaggageFields provisionExtra() {
+  @Override public BaggageFields create() {
```

Comment:
note: this is public just because the type itself lives in a different package (brave.internal.baggage) than the call site (brave.baggage). In most cases, they will be in the same package, so implementations can leave it protected.